CMakeLists.txt

```diff
@@ -49,6 +49,13 @@ if(ROBOTOLOGY_USES_MATLAB)
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/template/startup_robotology_superbuild.m.in ${CMAKE_BINARY_DIR}/startup_robotology_superbuild.m)
 endif()
 
+if(ROBOTOLOGY_USES_PYTHON)
+  find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+  execute_process(COMMAND ${Python3_EXECUTABLE} -c "from distutils import sysconfig; print(sysconfig.get_python_lib(1,0,prefix=''))"
+     OUTPUT_VARIABLE _PYTHON_INSTDIR)
+  string(STRIP ${_PYTHON_INSTDIR} ROBOTOLOGY_SUPERBUILD_PYTHON_INSTALL_DIR)
+endif()
+
```

Comment:
Checking again, shouldn't these lines go above `include(RobotologySuperbuildLogic)`?

Reply:
It should not make a big difference, no `Build<>.cmake` module actually searches for Python, the just call the `YCMEpHelper` module, but the actual configuration of the project is happening during superbuild build time, not during configuration time (in a nutshell: `cmake <src_dir>` for the subprojects is called when you run `make`, not when you run `cmake <src_dir>` for the superbuild).

Reply:
I'm experimenting in [`feature/gym-ignition`](https://github.com/robotology/robotology-superbuild/tree/feature/gym-ignition) branch and it might be necessary here:

https://github.com/robotology/robotology-superbuild/blob/fa9fdb7ea429eb84e03cbd4d7ced5344866722b3/cmake/Buildgym-ignition.cmake#L18

Reply:
Yes, in that case we need to move it before.

Reply:
Fixed.